miden/src/lib.rs

```diff
@@ -10,9 +10,9 @@ pub use assembly::{
     diagnostics, Assembler, AssemblyError,
 };
 pub use processor::{
-    crypto, execute, execute_iter, utils, AdviceInputs, AdviceProvider, AsmOpInfo, DefaultHost,
-    ExecutionError, ExecutionTrace, Host, Kernel, MemAdviceProvider, Operation, Program,
-    ProgramInfo, StackInputs, VmState, VmStateIterator, ZERO,
+    crypto, utils, AdviceInputs, AdviceProvider, AsmOpInfo, DefaultHost, ExecutionError,
+    ExecutionTrace, Host, Kernel, MemAdviceProvider, Operation, StackInputs, VmState,
+    VmStateIterator, ZERO,
```

Comment:
What's the reason for removing `execute`, `execute_iter`, and other previously exported members from here?

Reply:
Unintentional; fixed